### ISSUE TITLE
#coordinates parser function conflicts with GeoData extension

### ISSUE THREAD

**Issue by paulwalko** [JeroenDeDauw/Maps#530]
### Setup

- Maps version: 6.3
- MW version: 1.31
- PHP version: 7.2
- SMW version (if applicable): 

### Issue

I use the GeoData extension for the Nearby feature and this Maps extension for viewing maps. Both extensions define the #coordinates parser functions, and the Maps extension seems to override the GeoData extension. Here is an example of the error I see, from https://vtluug.org/wiki/Installfest_2016-02-14

```
Fatal error: The format of the coordinate could not be determined. Parsing failed.
```

I understand that this isn't a bug with the Maps extension itself, but I don't know of any other way to get around the issue. Others have had the issue as well: https://www.mediawiki.org/wiki/Topic:Tlck69vh90gav1er

Steps to reproduce the observation (recommendation is to use the [sandbox](https://sandbox.semantic-mediawiki.org)):

Install GeoData and Maps extensions and use the #coordinates in the syntax GeoData expects, such as {{#coordinates:37.230512|-80.421960|primary}}.

**Comment by JeroenDeDauw** [comment]
The maps extension could provide a configuration option to disable its coordinates function. Would that work for your usecase? Can you think of something else that would be better, either for your usecase or users of the extension(s) in general?

**Comment by paulwalko** [comment]
I'm gonna go ahead and close this since it seems like the most you can do here is a workaround that only works in my specific scenario. I've opened an issue with the ParserFunctions extension, which might be able to provide the functionality I'm looking for.

**Comment by JeroenDeDauw** [comment]
I do not think this can be fixed by the ParserFunctions extension. MediaWiki itself could perhaps provide a way to namespace parser functions. However since the conflict you are running into has happened for other people, since it is just this conflict that is likely to happen, and since it seems easy to fix, I think it is worth providing a setting to disable the maps coordinates parser function.

**Comment by paulwalko** [comment]
Good point, I was leaning away from doing that because having a variable to disable a the function seems like it could get messy in some that haven't been discovered yet. However, I was appreciate it very much if you added this feature!

Here's the thread from MW: https://phabricator.wikimedia.org/T212453. You're right that the ParserFunctions extension is the wrong place to implement this (my bad for not researching stuff enough). Unfortunately there's no system in place to implement this easily currently.

**Comment by JeroenDeDauw** [comment]
Done by https://github.com/JeroenDeDauw/Maps/commit/33db09cf317fcfc53295c58554a66c727a2cacec, which will be part of the 7.1 release

**Comment by JeroenDeDauw** [comment]
FYI, this has now been released as part of 7.1 https://github.com/JeroenDeDauw/Maps/blob/master/RELEASE-NOTES.md#maps-710

**Comment by Seb35** [comment]
And documented on https://www.semantic-mediawiki.org/wiki/Maps/Coordinates#Wiki_configuration.
